### ISSUE TITLE
Clicking on month view - selects completely different day

### ISSUE THREAD

**Issue by piernik** [fullcalendar/fullcalendar#4608]
In moth view I'm clicking on a specific day to add an event and fullcalendar pick completely different day (which is below picked date). For now I cannot reproduce.
Is anyone facing the same issue? 
I'm using `4.0.1`

**Comment by piernik** [comment]
I'm adding gif
![event-click](https://user-images.githubusercontent.com/420691/55553594-e370e300-56e0-11e9-858f-2395e147cfa4.gif)

**Comment by acerix** [comment]
It sounds like you need to call `updateSize()` after the calendar is displayed, for example, in the `shown.bs.modal` callback if it's Bootstrap.

Otherwise try turning off CSS and scripts and submit a [runnable, stripped-down demonstration of the bug](http://fullcalendar.io/wiki/Reduced-Test-Cases/) if there is still an issue.

**Comment by alexpotter** [comment]
I get the same issue, when the page first renders on any view, month, week or day.

If I change the view from month to week, back to month, it selects the correct date. Also when my component renders if I do the following

```ts
this.calendar.render()

setTimeOut(() => this.calendar.updateSize(), 500)
```

The issue does disappears, however does mean I now have a re render on my component. It seems the initial render doesn't work as expected when calling `this.calendar.render()`

**Comment by arshaw** [comment]
even tho there is no recreatable test case here, i think i know what's behind it. related to #4616 i believe

**Comment by arshaw** [comment]
this is fixed in v4.1.0
